help-docs/FAQ/join-tg-group.md

```diff
@@ -36,38 +36,46 @@ For best results:
 
 1. Verify you received a legitimate invite link from `@collablandbot`.
 
-   - The invite will be a link in the format telegram.me/**collablandbot**?start=ABCD1234...
+   - Your admin will share the invite will be a link in the format telegram.me/**collablandbot**?start=ABCD1234...
    - If the link is not _**exactly**_ `collablandbot`, it is an imposter.
 
 2. Use the invite link generated by the bot.  
 
-3. Verify the `Community:` field is correct for the community you are trying to join.
+3. Click the `START` button
+
+4. Verify the `Community:` field is correct for the community you are trying to join.
 
 <div class="text--center">
   <img src={img3a} alt="Verify Community details" width="400" />
 </div>
 
-4. Click the "Connect Wallet" button from the bot.
+5. Click the "Connect Wallet" button from the bot.
 
 <div class="text--center">
   <img src={img3} alt="Connect Wallet using `@collablandbot`" width="400" />
 </div>
 
-5. In your browser popup, select a wallet from the list of [Collab.Land-support wallets](/help-docs/wallets/verify-your-wallet#supported-wallets) to verify.
+:::caution
+
+You must complete the wallet verification process within 5 minutes or you will get an error. To resolve the error, start again from the "Connect Wallet" button in Telegram.
+
+:::
+
+6. In your browser popup, select a wallet from the list of [Collab.Land-support wallets](/help-docs/wallets/verify-your-wallet#supported-wallets) to verify.
 
 <div class="text--center">
   <img src={img4} alt="Select your wallet to verify" width="300"/>
 </div>
 
-6. Follow the instructions to have your wallet sign the connection request from the bot.
+7. Follow the instructions to have your wallet sign the connection request from the bot.

```

Comment:
regarding this, we have some good language here too https://docs.collab.land/help-docs/wallets/verify-your-wallet, except this line "This step is necessary to confirm your identity and ensure the security of your funds. After the first time, you will have the option to choose an existing connected wallet without having to sign a new message."

Reply:
something we mention in support:

Collab.Land is a read-only application. When you sign a message, you attest that you own that particular wallet address. Collab.Land reads the public blockchain to verify that the wallet address is connected to the required token address to grant a role. There is no opportunity for Collab.Land to have access to anything other than your public wallet address, which is viewable to everyone.

Reply:
> something we mention in support:
> 
> Collab.Land is a read-only application. When you sign a message, you attest that you own that particular wallet address. Collab.Land reads the public blockchain to verify that the wallet address is connected to the required token address to grant a role. There is no opportunity for Collab.Land to have access to anything other than your public wallet address, which is viewable to everyone.

except we're adding write capabilities with gmPASS

Reply:
added it but we'll have to change that very soon